lib/libzpool/Makefile.am

```diff
@@ -12,11 +12,16 @@ AM_CFLAGS += $(NO_UNUSED_BUT_SET_VARIABLE)
 # Includes kernel code generate warnings for large stack frames
 AM_CFLAGS += $(FRAME_LARGER_THAN)
 
-AM_CFLAGS += -DLIB_ZPOOL_BUILD
+AM_CFLAGS += -DLIB_ZPOOL_BUILD -Wframe-larger-than=32768
```

Comment:
then you have never compiled zstd :-) this is required, otherwise you get alot of warnings from the zstd code. the warnings i found are in the 20k range

Reply:
i removed now alot of dead and unused code from zstd lib, so this warning option is not required anymore and has been removed with latest commit